packages/jsActions/nanoflow-actions-native/src/geolocation/RequestLocationPermission.ts

```diff
@@ -29,14 +97,26 @@ export async function RequestLocationPermission(): Promise<boolean> {
                           status => status === PermissionsAndroid.RESULTS.GRANTED
                       )
             );
-        } else if (navigator.geolocation && (navigator.geolocation as GeolocationStatic).requestAuthorization) {
+        } else if (navigator.geolocation && (navigator.geolocation as any).requestAuthorization) {
             try {
-                (navigator.geolocation as GeolocationStatic).requestAuthorization();
+                (navigator.geolocation as any).requestAuthorization();
                 return Promise.resolve(true);
             } catch (error) {
                 return Promise.reject(error);
             }
         }
+
+        return false;
+    };
+
+    if (navigator && navigator.product === "ReactNative" && !navigator.geolocation) {
+        (navigator.geolocation as any) = NativeModules.RNFusedLocation ? RNGeolocationService : Geolocation;
```

Comment:
Is there a better type to use than `any`, what about RNGeolocationService | Geolocation's types?